pyndustri.pyi

```diff
@@ -52,10 +52,58 @@ class Env:
         Underscores become dashes (e.g. `Env.titanium_conveyor` becomes `@titanium-conveyor`).
         """
 
+class World:
+    """
+    World processor specific items
+    """
+
+    @staticmethod
+    def fetch(type: str, team: str, index: int):
+        """Fetches the item at a certain index. Type: {unit|player|core}."""
+    @staticmethod
+    def count(type: str, team: str) -> int:
+        """Get the thing count of a team, where thing: {unit|player|core}."""
+    @staticmethod
+    def fetchblock(type: str, team: str, index: int):
```

Comment:
Most multi-word defs are separated by `_`, but not here (and in a few others). Curious about the rationale.

Reply:
Hmm, i wasnt really thinking about it.